src/app/pages/details/details.js

```diff
@@ -0,0 +1,76 @@
+import BaseView from '~/helpers/backbone/view';
+import BaseModel from '~/helpers/backbone/model';
+import {props} from '~/helpers/backbone/decorators';
+import {template} from './details.hbs';
+import GetThisTitle from '~/components/get-this-title/get-this-title';
+
+@props({
+    model: new BaseModel(),
+    template: template,
+    templateHelpers: function () {
+        return this.model.get('bookInfo');
+    },
+    regions: {
+        getThisTitle: '.get-this-book'
+    }
+})
+export default class Subjects extends BaseView {
+    constructor() {
+        super(...arguments);
+        let title = decodeURIComponent(window.location.search.substr(1));
+
+        this.getThisTitle = new GetThisTitle(title);
+        this.model.set('bookInfo', {
+            title,
+            blurb: `College Physics meets standard scope and sequence requirements for a
+                two-semester introductory alegabra-based physics course. The text is
+                grounded aliqua salami tongue fugiat anim. Andouille sunt labore sint,
+                incididunt turkey tri-tip swine pork chop jowl cow fatback spare ribs shank
+                enim. Commodo cow laborum ad adipisicing spare ribs excepteur.`,
+            mainAuthorTitle: 'Senior Contributing Authors',
+            mainAuthorHtml: `
+                 <p>Dr. Paul Peter Urone, California State University Sacramento</p>
+                 <p>Dr. Roger Hinrichs, State University of New York, College at Oswego</p>
```

Comment:
In the design review, some things are plain text (like the author titles) and some things allow for HTML editing because the structure isn't well-defined in advance.